src/ripple/app/misc/ValidatorList.h

```diff
@@ -309,7 +309,8 @@ class ValidatorList
         std::function<void(PublicKey const&, bool)> func) const;
 
     static std::size_t
-    calculateQuorum (std::size_t nTrustedKeys);
+    calculateMinimumQuorum (
```

Comment:
Your call.  If you are comfortable getting coverage via that other spot, seems fine to me.